pages/Transfer.qml

```diff
@@ -772,7 +766,7 @@ Rectangle {
         }
 
         // There are sufficient unlocked funds available
-        if(parseFloat(amountLine.text) > parseFloat(middlePanel.unlockedBalanceText)){
+        if(parseFloat(amountLine.text) > parseFloat(walletManager.displayAmount(currentWallet.balance(currentWallet.currentSubaddressAccount)))){
```

Comment:
`currentWallet` might not be defined yet when opening the wallet. Consider adding something like this:

```
diff --git a/pages/Transfer.qml b/pages/Transfer.qml
index 47ec9c4..42a5c15 100644
--- a/pages/Transfer.qml
+++ b/pages/Transfer.qml
@@ -758,6 +758,9 @@ Rectangle {
         // reset message
         root.sendButtonWarning = "";
 
+        if (!currentWallet)
+            return true;
+
         // Currently opened wallet is not view-only
         if(appWindow.viewOnly){
             root.sendButtonWarning = qsTr("Wallet is view-only and sends are not possible. Unless key images are imported, " + 
```